### ISSUE TITLE
[Question] Constantly "Loading symbols for XX ..."

### ISSUE THREAD

**Issue by vlatkoB** [iquiw/company-ghc#16]
Hi,
`company-ghc` is constantly `Loading symbols for XX ...` while I type. Always the same modules (below). 

`company-ghc-diagnose`:
```
* company-ghc backend found: (company-ghc :with company-dabbrev-code)
* automatic scan module is enabled
* ghc-boot process has been done

Module					Alias		    Candidates
-------------------------------------------------------------------------------
Types.Plan				-			 nil
Types.MobNet				-			 nil
Types.Invoice				-			 nil
Types.Event				-			 nil
StdImports				-			 1250
MobNets.Extract				-			 nil
Prelude					-			 212
```
diagnose shows several modules with `nil` for candidates. Is it my config?

br,
vlatko

**Comment by iquiw** [comment]
It can happen if `(ghc-load-module MODULE)` returns non-list result.
There may be some error in `*GHC Debug*` buffer if `ghc-debug` variable is set to `t`.

I have not decided how to handle this case yet.

**Comment by iquiw** [comment]
"Loading symbols" are not shown from `company-ghc` now since it does not use `(ghc-load-module MODULE)` since v1.0.0. The fundamental problem is not solved though (#23).

As for `nil`, it is perhaps `ghc-mod`'s issue. Please file new issue if there is any problem in `company-ghc`.
